src/build/get_ios_deployment_target.mjs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {parseXmlFile} from './parse_xml_file.mjs';
+
+export async function getIosDeploymentTarget() {
+  const {
+    widget: {platform: platformList},
+  } = await parseXmlFile('config.xml');
+
+  const iosDeploymentTargetXMLTag = platformList
+    .find(({$}) => $.name === 'ios')
+    .preference.find(({$}) => $.name === 'deployment-target');
+
+  return `iPhone ${Number(iosDeploymentTargetXMLTag.$.value)}`;
```

Comment:
The deployment target is a number, not a string. I found this confusing. Let's just return the number and build the string in the caller.